### ISSUE TITLE
Support for Persona 5: Royal?

### ISSUE THREAD

**Issue by tSlappy** [esperknight/CriPakTools#13]
Is it possible to support newer .cpk format?

![image](https://user-images.githubusercontent.com/1177940/202766759-f932be86-7bd5-4ae3-af56-d7d538b54dff.png)

I can see the context of the archive, but extracting fails : Index was outside the bounds of the array.

**Comment by pixelkat5** [comment]
Yeah I got the same thing. it just crashes for me. I remember a tool that was just a fork of this but with support for p5r but it was on some mod website that I cant find.

**Comment by pixelkat5** [comment]
https://github.com/Sewer56/CriFsV2Lib/releases/tag/2.1.2
